src/utils/add_locations_metadata.R

```diff
@@ -0,0 +1,43 @@
+box::use(dplyr)
+box::use(stringr)
+
+box::use(cs = ./cloud_storage)
+
+#' Add location and region columns
```

Comment:
feel like it would be nice to add a "why" part to the explanation. And maybe that would include where the function is called

Reply:
Also I think IRN `location` looks truncated: 

`Iran (Islamic Republic of)`

Reply:
In the maps?